src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,13 @@
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 24px;
+}
+
+.rating__review {
+  font-size: 10px;
+  font-weight: 400;
+  color: $standart-black-color;
```

Comment:
The variable '$standart-black-color' also has a typo. Consider renaming it to '$standard-black-color' for consistency and correct spelling.